### ISSUE TITLE
simOTA calculate cost

### ISSUE THREAD

**Issue by Flyingdog-Huang** [Megvii-BaseDetection/YOLOX#1600]
https://github.com/Megvii-BaseDetection/YOLOX/blob/5c110a25596ad8385e955ac4fc992ba040043fe6/yolox/models/yolo_head.py#L478

why use BCE instead of BCE with_logits?

**Comment by fengkh** [comment]
Because the input of this method (BCE) already passed Sigmoid layer, dont need BCE_with_logits .
